src/utils.ts

```diff
@@ -6,3 +6,63 @@
 export function parseCoords(coords: string): number[] {
     return coords.split(/\s?,\s?/).map((c: string) => parseFloat(c ?? 0))
 }
+
+export enum Keys {
```

Comment:
maybe it's not the best idea to add enums to the `utils` module? I guess it will be better to create new file with enums in some `data` folder